SH4RE/SH4RE/CreateListing/CreateListingView.swift

```diff
@@ -214,20 +214,23 @@ struct CreateListingView: View {
     }
     private var availabilityView: some View {
         Group {
-            Text("Availability")
-                .font(.title2)
-                .frame(maxWidth: screenSize.width * 0.9, alignment: .leading)
-                .padding(.bottom)
-            
-            // availability
-            DropdownMenu(label: "Availability for the Next 3 Months", options: availabilityList, selection: $availabilitySelection)
-                .frame(maxWidth: screenSize.width * 0.9)
-                .onChange(of: availabilitySelection) { value in
-                    availabilityCalendar.selectedDates = []
-                }
-            
+            if(!isEditing){
+                Text("Availability")
+                    .font(.title2)
+                    .frame(maxWidth: screenSize.width * 0.9, alignment: .leading)
+                    .padding(.bottom)
+                
+                // availability
+                DropdownMenu(label: "Availability for the Next 3 Months", options: availabilityList, selection: $availabilitySelection)
+                    .frame(maxWidth: screenSize.width * 0.9)
+                    .onChange(of: availabilitySelection) { value in
+                        availabilityCalendar.selectedDates = []
+                    }
+            }
```

Comment:
got rid of allowing weekend, weekdays for editing availability since logic became too complicated when making sure we don't overwrite bookings